scripts/release/build/rpm/package.sh

```diff
@@ -1,30 +1,34 @@
-#!/bin/bash -e
-
-# build_rpm.sh - Build a .deb package for one platform.
-#
-# Syntax:   build_rpm.sh <output directory>
-#
```

Comment:
Consider specifying environment variables that are either required or optional.

Reply:
There aren't any.

Reply:
There are some shell variables that are inherited from parent processes.  One of the first tasks in `setup.sh` for almost every Jenkinsfile is creating a `build_env` file that is then often sourced by parent processes.

The `mule` tasks won't work like this.